tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+retrying
```

Comment:
maybe we should call this file - bootstrap-sanity-requirements.txt to make it clearer what it relates to